### ISSUE TITLE
Electrum 2.6,2 Error: 'NoneType' object has no attribute '__getitem__'

### ISSUE THREAD

**Issue by njamtf** [spesmilo/electrum#1721]
This error is popping up on both my Electrum 2.6.2 client on Mac OS Yosemite as well as on Electrum 2.5.4 client on Tails 2.2 whenever I try to send bitcoin to another address. Please help!

**Comment by ecdsa** [comment]
what type of wallet is that?
is there a complete traceback in your terminal?

**Comment by ecdsa** [comment]
why did you close the issue?

**Comment by njamtf** [comment]
Sorry, it turns out the mistake was incredibly simple. I just logged out of
both accounts and recovered on the 2.6.2. When I tried the transfer funds
again, it asked for a password which it hadn't been doing before --
presumably because I was in a watch-only mode (which was never indicated
after an initial recovery). It was legit just a dumb user error fixed by
the equivalent of turning the power off and flipping it back on again.

Regardless, thanks for replying to my inquiry. I closed the issue because I
felt dumb.
